Lib/glyphsLib/builder/masters.py

```diff
@@ -90,6 +90,13 @@ def to_glyphs_master_attributes(self, source, master):
         # GSFontMaster has a random id by default
         pass
 
+    if master.id.lower() in (m.id.lower() for m in self.font.masters):
+        raise ValueError(
+            "{} contains a '{}' lib key with the duplicate value '{}'. All given masters must have a unique ID or data will get corrupted. Please check for this lib key and either remove it (will be regenerated) or change the value. If there is no key, you just witnessed something unlikely.".format(
```

Comment:
you should split these long lines manually, black doesn't help there

Reply:
This is something that black should have one day, I need it constantly 😁